### DISCUSSION TITLE
Miért nem az applikációban lehet megtekinteni a ManageDeclarationSubmissionResponse-ban küldött bevallást?  Ez miért kliens oldali feladat?

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#38] [Kérdések és válaszok - Gépi interfész]
Kedves NAV Szakértők!

Miért nem az applikációban lehet megtekinteni a ManageDeclarationSubmissionResponse-ban küldött bevallást?  Ez miért kliens oldali feladat?

A választ előre is köszönjük!

**Comment by renced42** [comment]
Kedves @szalaierika 

Nem teljesen értem a kérdést. Milyen applikációra gondolsz? 
Ha az eÁfa webes felületére, akkor ott meg lehet majd nézni a beadott bevallást.
A kliens oldalon ismertek az összeállított adatok ott azt mutat meg a felhasználónak ami a kliens oldali igény.  Illetve a bevallást is le lehet majd tölteni.

**Reply by szalaierika** [reply]
Kedves @renced42 ,

a doksi 15. oldalán 2. pontban leírt mobilapplikációra gondoltam:

Az áfabevallás XML jóváhagyása két lépcsőben történik:
1. Az áfabevallás XML jóváhagyását a ManageDeclarationSubmission operáción keresztül lehet elindítani.
2. Elsődleges felhasználónak egy társított mobilapplikáción keresztül szükséges a jóváhagyásról döntést hoznia.

**Reply by NTCA-HM** [reply]
Kedves @szalaierika !

Az áfabevallás XML API-n keresztül hagyható jóvá. A GitHub-ra még februárban kihelyezett folyamatleírásban (15. oldal) foglaltakkal ellentétben, mégsem lesz szükség mobilapplikációra a bevallás jóváhagyásához. A véglegesített eÁFA M2M követelményjegyzék már így tartalmazza a jóváhagyási folyamat leírását.

**Reply by BoschHU** [reply]
Kedves @NTCA-HM !
Fejlesztőink a májusban publikált angol nyelvű specifikáció alapján dolgoznak. Ebben még mobilapplikáció szerepel a bevallás jóváhagyásához (2.4.3). Kérhetjük, hogy csak a még fennálló szabályok legyenek itt elérhetőek, és hogy angol nyelvre a legutolsó verzió is legyen lefordítva (beleértve az interface specifikációkat)?

**Reply by NTCA-HM** [reply]
Kedves @BoschHU !

A specifikáció átdolgozása folyamatban van, és így annak angol nyelvű fordítása is. Amint elkészül itt a GitHub-on publikálni fogjuk, addig is kérjük szíves türelmüket!

**Reply by szalaierika** [reply]
Kedves @NTCA-HM !

Köszi szépen a választ. Az új specifikációban nem találunk olyan operációt, amivel jóváhagyható a bevfelddata XML. Melyik ez az operáció és hol találjuk a jóváhagyási lépés leírását a dokumentációban?

**Comment by renced42** [comment]
Kedves @szalaierika 

Az eredeti kérdésre vonatkozóan a bevallás benyújtása a A /manageDeclarationSubmission az áfabevallás benyújtására szolgáló operáció. a specifikáció 3.3.5-ös pontja alapján.
